components/nft-sale-widget/layout/mint-random-section.tsx

```diff
@@ -1,34 +1,37 @@
-import NFTPrice from "@/components/commons/nftprice";
-import Divider from "@/components/divider";
-import MintButton from "@/components/mint-button";
-import { Box, Flex, Image } from "@chakra-ui/react";
+import { Box, Flex } from "@chakra-ui/react";
+import NFTItem from "../psyc-item";
+
+interface MintRandomSectionProps {
```

Comment:
If we have Token in other file, can we maybe have it in common files or something with interfaces? 

```ts
export interface TokenItem {
  src: string;
  price: string;
  isSold: boolean;
  batchId: string;
  tokenId: string;
}
```

Reply:
and have this as Token[] ?

Reply:
yeah.. I was going to do that when I refactor. Thanks for catching that 🙂